tests/ui/traits/const-traits/const-traits-core.rs

```diff
@@ -1,6 +1,6 @@
-//@ run-pass
```

Comment:
Please don't turn a run-pass test into a failing one. Please put the clone tests into a separate file.